src/SarifCli/SarifValidationContext.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.Cli
+{
+    public class SarifValidationContext : IAnalysisContext
+    {
+        public bool IsValidAnalysisTarget
+        {
+            get
+            {
+                return Path.GetExtension(TargetUri.LocalPath).Equals(".sarif", StringComparison.OrdinalIgnoreCase);
+            }
+        }
+
+        public IAnalysisLogger Logger { get; set; }
+
+        public string MimeType
+        {
+            get { return "text/x-sarif"; }
```

Comment:
We can't yet DRY out SarifCli's usage of this literal string -- that is, we can't have SarifCli refer to the new value of `MimeType.Sarif` -- until the next time we upgrade SarifCli to consume the latest version of the SDK. Remember, SarifCli consumes the SDK from NuGet, not as a P2P dependency.

---
In reply to: [77239179](https://github.com/Microsoft/sarif-sdk/pull/501#discussion_r77239179) [](ancestors = 77239179)